node/nodeRunner.go

```diff
@@ -676,23 +679,23 @@ func (nr *nodeRunner) logSessionInformation(
 	sessionInfoFileOutput string,
 	managedCoreComponents mainFactory.CoreComponentsHandler,
 ) {
-	configs := nr.configs
 	statsFolder := filepath.Join(workingDir, core.DefaultStatsPath)
+	configurationPaths := nr.configs.ConfigurationPathsHolder
 	copyConfigToStatsFolder(
 		statsFolder,
 		[]string{
-			configs.ConfigurationFileName,
-			configs.ConfigurationEconomicsFileName,
-			configs.ConfigurationRatingsFileName,
-			configs.ConfigurationPreferencesFileName,
-			configs.P2pConfigurationFileName,
-			configs.ConfigurationFileName,
-			configs.FlagsConfig.GenesisFileName,
-			configs.FlagsConfig.NodesFileName,
-			configs.ConfigurationApiRoutesFileName,
-			configs.ConfigurationExternalFileName,
-			configs.ConfigurationSystemSCFilename,
-			configs.ConfigurationGasScheduleDirectoryName,
+			configurationPaths.MainConfig,
+			configurationPaths.Economics,
+			configurationPaths.Ratings,
+			configurationPaths.Preferences,
+			configurationPaths.P2p,
+			configurationPaths.MainConfig,
```

Comment:
This config is also added in first index

Reply:
removed